.travis.yml

```diff
@@ -6,6 +6,8 @@ branches:
     - master
 
 script:
+  # Make sure there are no left-over git patch markers in
+  - npm run ci-checks
```

Comment:
The comment "# Make sure there are no left-over git patch markers in" looks much less generic than the name of the command. Is it the intention to add more tests into `ci-checks`, and if so will that comment age badly?